belvedere/src/main/java/zendesk/belvedere/ImageStream.java

```diff
@@ -217,6 +226,11 @@ public boolean isAttachmentsPopupVisible() {
          * The user deselected one or multiple attachments.
          */
         void onMediaDeselected(List<MediaResult> mediaResults);
+
+        /**
+         * The user used the send images button
+         */
+        void onMediaSent(List<MediaResult> mediaResults);
```

Comment:
Let's create another listener instead of extending this one to avoid breaking API change?
Example: scrolling: https://github.com/zendesk/belvedere/blob/af57cdc5b113cb9a0f290bbac1d1e9b22c45033d/belvedere/src/main/java/zendesk/belvedere/ImageStream.java#L176